src/tests/advanced.js

```diff
@@ -192,44 +178,14 @@ describe('Advanced', function () {
       //    100       USD
       //  +   4.99    USD (money from Alice)
       //  -   0.00998 USD (mark: spread/fee)
-      //  -   0.00498 USD (mark: quoted connector slippage; (4.99 - 0.00998) * 0.001 = 0.00498002)
       //  ===============
-      //    104.97504 USD
-      //    104.9750  USD (round destination amount down)
-      yield services.assertBalance('test2.ledger1.', 'bob', '104.975')
-      yield services.assertBalance('test2.ledger1.', 'mark2', '995.025')
-      yield graph.assertZeroHold()
+      //    104.98002 USD
+      //    104.9800  USD (round destination amount down)
+      yield services.assertBalance('test2.ledger1.', 'bob', '104.98')
+      yield services.assertBalance('test2.ledger1.', 'mark2', '995.02')
     })
 
-    it('zero slippage', function * () {
```

Comment:
Sorry, I should have said - this PR is based on top of DJ's unmerged changes. My commit is only the second one